recipes/consul_template.rb

```diff
@@ -42,7 +42,13 @@
   action :create
 end
 
-%w(/etc/nginx/template.ctmpl /etc/nginx/template.cfg /etc/nginx/template.watch /etc/nginx/template.kv.watch).each do |tmpl|
+%w(
+    /etc/nginx/template.ctmpl
+    /etc/nginx/templateSSL.ctmpl
+    /etc/nginx/template.cfg
+    /etc/nginx/template.watch
+    /etc/nginx/template.kv.watch
+  ).each do |tmpl|
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.